### PR TITLE
Fix rendering of on-page debug logs in modern ui

### PR DESCRIPTION
**Pull request by syncguru**
Fixes #22672

Here is what it looks like after applying this fix:

screen shot 2017-04-05 at 2 30 44 pm

### CODE REVIEW

File: core/layout_api.php [hunk 1]
```diff
@@ -1008,6 +1008,9 @@ function layout_page_content_begin() {
  * @return null
  */
 function layout_page_content_end() {
+    # Print table of log events
```
Comment (vboctor) [inline core/layout_api.php]: Please fix indentation.

File: core/logging_api.php [hunk 2]
```diff
@@ -183,21 +183,36 @@ function log_print_to_page() {
 		$t_total_queries_count = 0;
 		$t_total_event_count = count( $g_log_events );
 
-		echo "\t<hr />\n";
+
+        echo "<div class=\"space-10\"></div>";
```
Comment (vboctor) [inline core/logging_api.php]: Indentation

**Comment by cproensa** [comment]
Functionally, i see it ok.

**Comment by syncguru** [comment]
@vboctor fixed indentations

**Pull request merged**